lib/secure_headers/headers/public_key_pins.rb

```diff
@@ -1,23 +1,37 @@
 module SecureHeaders
-  class PublicKeyPinsBuildError < StandardError; end
+  class PublicKeyPinsConfigError < StandardError; end
   class PublicKeyPins < Header
-    module Constants
-      HPKP_HEADER_NAME = "Public-Key-Pins"
-      ENV_KEY = 'secure_headers.public_key_pins'
-      HASH_ALGORITHMS = [:sha256]
-      DIRECTIVES = [:max_age]
-      CONFIG_KEY = :hpkp
-    end
+    HEADER_NAME = "Public-Key-Pins".freeze
+    HASH_ALGORITHMS = [:sha256].freeze
+    DIRECTIVES = [:max_age].freeze
+    CONFIG_KEY = :hpkp
```

Comment:
All of these CONFIG_KEYS will be scoped to https://github.com/twitter/secureheaders/pull/181/files#diff-e7fcd10663004caed933b95e0ff2e643R23 in https://github.com/twitter/secureheaders/pull/181/files#diff-e7fcd10663004caed933b95e0ff2e643R160. That will be "unsupported" when I remove the line from your comment about pulling the config from two different places in `request.env`. 

Unfortunately that's a left over from when I thought passing a hash directly to `header_hash_for` would be a thing. :hankey: